security/4.0/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Jakarta Security 4.0 (under development)"
+date: 2023-05-30
+summary: "Release for Jakarta EE 11"
+---
+Jakarta Security defines a standard for creating secure Jakarta EE applications in modern application paradigms.
+
+The release splits focuses on evolving the API in various ways, and specifically providing APIs for the authorization theme (including interceptors and an abstraction for the permission store)
+
+### New features, enhancements or additions
+* Client-cert and Digest SECURITY #120 
+* Authentication mechanism per URL SECURIY #86 
```

Comment:
Non-blocking comments:

Again, should `#86` be a link? On the preview it's just text and doesn't point to the issue.

Also I assume it should say `SECURITY` rather than `SECURIY`?

Reply:
```suggestion
* Authentication mechanism per URL SECURITY #86 
```